src/ray/raylet/raylet_client.h

```diff
@@ -145,6 +161,24 @@ class RayletClient {
   /// \return ray::Status.
   ray::Status FreeObjects(const std::vector<ray::ObjectID> &object_ids, bool local_only);
 
+  /// Subscribe the object ready event.
+  ///
+  /// \param subscription_id The subscription ID.
+  /// \param object_id The event body (the ObjectID to be subscribed).
+  /// \return ray::Status.
+  ray::Status SubscribeObjectLocalEvent(const ray::ObjectID &subscription_id,
```

Comment:
don't use `ObjectID` for things that are not object IDs, this applies in a bunch of places